devito/dse/rewriters.py

```diff
@@ -382,6 +383,7 @@ def _eliminate_inter_stencil_redundancies(self, cluster, template, **kwargs):
 class AggressiveRewriter(AdvancedRewriter):
 
     def _pipeline(self, state):
+        self._new_pass(state)
```

Comment:
what is this?

Reply:
I need a dummy name as long as I do not have a real-existing pass.

Reply:
How should I call it?

Reply:
since it's unused and basically a no-op, it shouldn't appear in this PR